.vscode/cspell.json

```diff
@@ -415,7 +415,20 @@
       ]
     },
     {
-      "filename": "sdk/communication/azure-communication-identity/tests/**",
+      "filename": "sdk/cognitivelanguage/azure-ai-language-conversations/**",
+      "words": ["conv"],
+      "caseSensitive": false
+    },
+    {
+      "filename": "sdk/cognitivelanguage/azure-ai-language-conversations/doc/*.rst",
+      "words": ["undoc"]
+    },
+    {
+      "filename": "sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/**/_serialization.py",
+      "words": ["ctxt", "wday", "mday", "astimezone"]
```

Comment:
I like the approach of just ignoring the misspelled words in the _serialization.py files! Can we completely revert edits to the _serialization files (moving any other misspelled words to this list)? Additionally, I think if we cover our bases here then we don't need to add the ignore on L36 of the cspell.json. How does that sound?

Reply:
Love it @kristapratico! I've deferred the typos to the overrides list as you've mentioned :)

> https://github.com/Azure/autorest.python/issues/1414 
Seems the misspelled words it complains about are
"ctxt", "wday", "mday", "astimezone", "unflattened", "JSONify", "deseralize"

I think the last 3 words can be changed to "stacked", "JSON-ify", "deserialize" later in autorest. As for the other words, when we find another occurance in a different library, we can move them to the global words list. Please let me know what you think of this @kristapratico  ❤️

Reply:
Sounds good! This PR looks good to me so I'm going to approve and merge now. Thanks again for all your contributions 🚀